src/core/datatablemodule.cc

```diff
@@ -187,10 +233,22 @@ static py::oobj in_debug_mode(const py::PKArgs&) {
 }
 
 
+static const char* doc_get_thread_ids =
+R"(
+Return system ids of all threads used internally by datatable.
+
+Calling this function will cause the threads to spawn if they
+haven't done already. (This behavior may change in the future).
+
+Parameters
+----------
+return: List[str]
+    The list of thread ids used by the datatable. The first element
+    in the list is the id of the main thread.
+)";
```

Comment:
May be add a "See also" section with a link to `dt.options.nthreads`? Though we first need to merge my PR then to test.

Reply:
I added the see also link, hopefully it'll resolve once your PR is merged